windows/src/global/vc/keymansentry.cpp

```diff
@@ -11,6 +11,7 @@
 bool g_report_exceptions, g_report_messages;
 
 int keyman_sentry_init(bool is_keyman_developer) {
+#if 0
```

Comment:
Is the `//KEYMAN-SENTRY:UNDO: `notation to make it easier to find all the places to undo later?

Reply:
Pretty much. Just a mental note for me.